src/main/java/com/iota/iri/network/pipeline/PreProcessStage.java

```diff
@@ -59,7 +62,18 @@ public ProcessingContext process(ProcessingContext ctx) {
         long txDigest = NeighborRouter.getTxCacheDigest(txDataBytes);
 
         Hash receivedTxHash = recentlySeenBytesCache.get(txDigest);
-        Hash requestedHash = HashFactory.TRANSACTION.create(reqHashBytes, 0, Protocol.GOSSIP_REQUESTED_TX_HASH_BYTES_LENGTH);
+        Hash requestedHash = HashFactory.TRANSACTION.create(reqHashBytes, 0,
+                Protocol.GOSSIP_REQUESTED_TX_HASH_BYTES_LENGTH);
+
+        // log cache hit/miss ratio every 50k get()s
+        if (log.isDebugEnabled()) {
+            long hits = recentlySeenBytesCache.getCacheHits();
+            long misses = recentlySeenBytesCache.getCacheMisses();
+            if ((hits + misses) % 50000L == 0) {
+                log.info("recently seen bytes cache hit/miss ratio: {}/{}", hits, misses);
```

Comment:
Change this to `log.debug` so it will be clear that this is only for debug mode please

Reply:
Changed to debug.